src/components/SidebarTabs/SharingSidebarTab.vue

```diff
@@ -91,6 +97,11 @@
 			</div>
 		</TransitionGroup>
 
+		<NcDialog :open.sync="qrDialog" close-on-click-outside="true" @close="qrDialog=false">
+			<QRDialog :title="t('forms', 'Share {formTitle}', { formTitle: form.title })"
+				:text="qrDialog" />
+		</NcDialog>
```

Comment:
```suggestion
		<QRDialog :open="qrDialogText !== null"
			:title="t('forms', 'Share {formTitle}', { formTitle: form.title })" 
			:text="qrDialogText"
			@close="qrDialogText = null" />
```

Abstract the dialog and move it to the component instead.